content/wiki/canonical_lints.md

```diff
@@ -68,15 +71,31 @@ clippy.wildcard_dependencies = "warn"
 # END LINEBENDER LINT SET
 ```
 
-And in their `lib.rs`:
+## `lib.rs`
 
 ```rust
-// LINEBENDER LINT SET - v1
+// LINEBENDER LINT SET - lib.rs - v1
```

Comment:
I don't think having separate versioning for each part helps anyone...

If the versioning is the same, it seems easier to search across the whole repository for an outdated version if we don't have the file name in the middle.

Reply:
Doing the search is easier, but the results won't actually be outdated. Like in this very PR the `lib.rs` set doesn't change and I imagine that to be the case in most version bumps. The `lib.rs` set is also the one that needs the most pasting as there are multiples of this per repo.

So separating the versions would certainly help me in not having to update those files.

So to be clear, you think we should have a unified version and also always update that header in repos even when no content changes?

Reply:
Hmm... I guess that makes sense. This is probably reasonable then.